pkg/wakaama/contrib/lwm2m_client.c

```diff
@@ -171,6 +171,9 @@ static void *_lwm2m_client_run(void *arg)
         sock_udp_ep_t local;
         int res = sock_udp_get_local(&_client_data->sock, &local);
         assert(res >= 0);
+#if IS_ACTIVE(NDEBUG)
+        (void)res;
+#endif
```

Comment:
No need for preprocessor conditionals here, casting to `void` has no disadvantage for the case with assertions enabled, so it can just always be done.